vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/bean/boutiquesTools/BoutiquesApplication.java

```diff
@@ -38,6 +41,8 @@ public class BoutiquesApplication implements IsSerializable {
     private Set<BoutiquesOutputFile> outputFiles = new HashSet<>();
     private Map<String, String> tags = new HashMap<>();
     private String jsonFile;
+    private Set<String> vipDotInputIds;
+    private String resultDirs;
```

Comment:
rename field and getter/setter with `vipDotIncludesResultsDir`, and change type to boolean.

Reply:
renamed to vipDotIncludesResultsDir